README.md

```diff
@@ -351,6 +351,11 @@ Textual Inversion is a technique for capturing novel concepts from a small numbe
 - Full Stable Diffusion fine-tuning. If you have a more sizable dataset with a specific look or style, you can fine-tune Stable Diffusion so that it outputs images following those examples. This was the approach taken to create [a Pokémon Stable Diffusion model](https://huggingface.co/justinpinkney/pokemon-stable-diffusion) (by Justing Pinkney / Lambda Labs), [a Japanese specific version of Stable Diffusion](https://huggingface.co/spaces/rinna/japanese-stable-diffusion) (by [Rinna Co.](https://github.com/rinnakk/japanese-stable-diffusion/) and others. You can start at [our text-to-image fine-tuning example](https://github.com/huggingface/diffusers/tree/main/examples/text_to_image) and go from there.
 
 
+## Quantizing Stable Diffusion
+
+Quantization is one of widey-used model compression techniques to accelerate the inference performance. There are two typical approaches: 1) post-training quantization (PTQ), and 2) during-training quantization (DTQ). PTQ is an effective approach to quantizing a model without additional training steps but requires an offline calibration process using a representative dataset to determinate the quantization parameters (e.g., scale and zero point) for the model. DTQ is designed to simulate the quantization error and get recovered as much as possible during training, and quantization-aware training is a well-known approach for DTQ. Optimum-Intel provides the quantization support and sample code on how to enable quantization for Stable Diffusion using [post-training static or dynamic quantization](https://github.com/huggingface/optimum-intel/tree/main/examples/neural_compressor/text-to-image).
```

Comment:
That works with me - should we maybe also add a Python example below:
```bash
install optimum intel stable diffusion
...
```

```python
# import optimum-intel stable diffusion
# run optimization
...
```

Also we could make a doc page below one of those:  https://huggingface.co/docs/diffusers/optimization/mps ?

Reply:
Thanks @patrickvonplaten for your comments. Seems there is some issue in my code sync in this PR so I created a new PR (https://github.com/huggingface/diffusers/pull/1324) to make PR clean. Sample code was added. Please help review from there.

BTW, could you please share the instructions on how to create a doc page?